torch/csrc/autograd/python_variable.cpp

```diff
@@ -376,6 +380,14 @@ int THPVariable_set_requires_grad(THPVariable *self, PyObject *obj)
   return 0;
 }
 
+PyObject *THPVariable_get_pretty_name(THPVariable* self)
+{
+  return self->cdata.pretty_name() == ""
+    ? NULL
```

Comment:
Do you want to return `None`? Use the `Py_RETURN_NONE` macro. Returning NULL usually signifies an error.